database/migrations/Inventory/2024_07_22_190318_attribute-type-migration.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class () extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('attributes_types_input', function (Blueprint $table) {
+            $table->id();
+            $table->string('name');
+            $table->string('slug')->nullable(true);
+            $table->bigInteger('apps_id')->unsigned();
```

Comment:
index for is_Deleted and do a index of apps + company also add the user_id so we know who created what